modules/tracking-the-compliance-status-of-your-environment.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-compliance.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="tracking-the-compliance-status-of-your-environment_{context}"]
+= Tracking the compliance status of your environment
+
+By generating compliance reports, you can keep a track of the compliance status of your environment. You can use these reports to convey compliance status across various industry mandates to other stakeholders.
+
+You can generate the following reports:
+
+* *Executive reports* that focus on the business aspect and include charts and a summary of the compliance status in PDF format.
+* *Evidence reports* that focus on the technical aspect and contain detailed information in CSV format.
+
+.Procedure
+
+. In the {product-title-short} portal, go to *Compliance -> Dashboard*.
```

Comment:
Thanks for the feedback. Although, the ISG encourages us to us "Click" in navigation menus I am bit confused about this usage. When we say, "go to" does it not imply that we have to click the menu item?
@bergerhoffer your thoughts, please, thx!

Reply:
I'm not sure whether the ISG actually favors "click" over "go to", but I know that we use "click" much more often. To me, "go to" doesn't sound quite right to the ear. Even though it's longer, another option that I think sounds better is "navigate to". 

However, in general it's usually better to avoid using two words when a one-word alternative would do.

So if it were me, I'd go either with "click" or "navigate to".